src/main/java/frc/utils/math/MirrorMath.java

```diff
@@ -18,4 +21,25 @@ public static double getMirroredY(double y) {
 		return Field.WIDTH_METERS - y;
 	}
 
+	public static Pose2d getMirroredPose(Pose2d pose2d, boolean invertY, boolean invertAngle) {
```

Comment:
```suggestion
	public static Pose2d getMirroredPose(Pose2d pose2d, boolean mirrorY, boolean invertAngle) {
```
I you have a mirrorY argument, why don't you have a mirrorX arg too?